src/mount/writedata.cc

```diff
@@ -703,6 +706,7 @@ void write_data_init(uint32_t cachesize, uint32_t retries, uint32_t workers,
 	pthread_attr_destroy(&thattr);
 
 	gTweaks.registerVariable("WriteMaxRetries", maxretries);
+	gTweaks.registerVariable("IgnoreFlush", gIgnoreFlush);
```

Comment:
I see this change does not include an integration test. Please, consider adding an integration test like `SanityChecks/test_mount_directio.sh` to verify that modifying `ignoreflush` option on the fly works correctly. 

Another idea can be to create a test specifically for options that can be tweaked. In that way, every time a new option is added to the tweaks variable, we add the respective check for the new variable.

Reply:
At the end, I decided to add that requested integration test to the test `SanityChecks/test_mount_tweaks.sh` which is a substitution for `SanityChecks/test_mount_directio.sh` for testing `directio` and `ignoreflush` variables.